### PR TITLE
biocollect-196 Associated organisations - presentation on about page

### PR DESCRIPTION
**Pull request by m-r-c**

### CODE REVIEW

File: grails-app/views/project/_aboutCitizenScienceProject.gsp [hunk 1]
```diff
@@ -119,7 +117,50 @@
...
 
             map = new MapWithFeatures(mapOptions, mapFeatures);
+        }
+        </g:if>
     }
-</g:if>
+
+    // make sure the list of associated organisations are below the shorter of the two columns.
+    function placeAssociatedOrgs() {
+        console.log("col1 = " + $("#column1").height() + "; col2 = " + $("#column2").height())
```
Comment (sat01a) [inline grails-app/views/project/_aboutCitizenScienceProject.gsp]: Pls remove console.log

File: grails-app/views/project/_aboutCitizenScienceProject.gsp [hunk 2]
```diff
@@ -119,7 +117,50 @@
...
     }
+
+    setTimeout(placeAssociatedOrgs, 2000);
 </r:script>
+
+
+
+<script type="text/html" id="associated-orgs">
+<div class="well span12 margin-left-0" id="associatedOrgs">
+    <div class="well-title"><g:message code="project.display.associatedOrgs"/></div>
```
Comment (sat01a) [inline grails-app/views/project/_aboutCitizenScienceProject.gsp]: Looks like the div will appear even when there is no associated orgs? Is that intentional?
screen shot 2015-11-05 at 10 27 19 am

Comment (m-r-c) [inline grails-app/views/project/_aboutCitizenScienceProject.gsp]: Fixed

File: grails-app/views/project/_aboutCitizenScienceProject.gsp (hunk 1 above)
Comment (m-r-c) [inline grails-app/views/project/_aboutCitizenScienceProject.gsp]: Done

**Pull request merged**